python/GafferUI/CompoundEditor.py

```diff
@@ -1661,8 +1663,35 @@ def getToolTip( self ) :
 			for d in drivenEditors :
 				toolTipElements.append( " - _%s_" % d.getTitle() )
 
+		self.__highlightRelatedEditors( editor, True )
+
 		return "\n".join( toolTipElements )
 
+	@staticmethod
+	def __highlightRelatedEditors( editor, highlighted ) :
+
+		toHighlight = []
+
+		masterEditor = editor.drivingEditor()
+		if masterEditor is not None :
+			toHighlight.append( masterEditor )
+		else :
+			toHighlight.extend( editor.drivenNodeSets( recurse = True ).keys() )
+
+		for e in toHighlight :
+			if e._qtWidget().isHidden() :
```

Comment:
I definitely think this highlighting behaviour is a great feature, but I'm finding the specifics of the logic here a bit confusing in practice. When the editor is the current tab, I'm seeing the tab outline highlighted as well as the container around it. And when the editor isn't the current tab, I'm seeing just the container highlighted, which is a little non-specific. How feasible is it to highlight the tab _labels_ for all active tabs, in addition to the container outline? That way you could see when multiple editors in a container are affected, and you can see _which_ editor is affected when it's not the current one.

Reply:
Alternatively, if we could highlight the tab outlines for all affected tabs, and leave the container alone, that would have a similar effect...

Reply:
> When the editor is the current tab, I'm seeing the tab outline highlighted as well as the container around it

It's actually a little different. You only see the dual outline when there is both the current tab, and a non-current tab both being highlighted (which they are in the default layout). If its just the current tab on its own, then you only see that one highlighted.   The alternative was to not highlight non-visible tabs, which is cleaner, but its confusing if it says its linked, then nothing gets highlighted as they're non-current.

> How feasible is it to highlight the tab labels for all active tabs, in addition to the container outline? 

Sadly with Qt its a real pain, the only pseudo-classes you have in CSS are current tab and 'the rest'. There is, to the best of my knowledge from my various endeavours in Qt of late, zero way to target individual, specific tabs.

Reply:
I was contemplating a fainter highlight for the outer container for non-visible tabs, which would look better, but then it would mean splitting the use of `setHighlighted` and manual style hackery, or introducing a new public highlight state that was pretty, (or adding `secondary=True` or something to `setHighlighted`.

Reply:
> Sadly with Qt its a real pain, the only pseudo-classes you have in CSS are current tab and 'the rest'. There is, to the best of my knowledge from my various endeavours in Qt of late, zero way to target individual, specific tabs.

I suspected that might be the case. I was wondering if we could monkey with the tab label in some way to highlight the text, but maybe it's not worth the bother right now.

> I was contemplating a fainter highlight for the outer container for non-visible tabs, which would look better, but then it would mean splitting the use of setHighlighted and manual style hackery, or introducing a new public highlight state that was pretty, (or adding secondary=True or something to setHighlighted.

If we wanted to add API for this, I think `TabbedContainer.setHighlighted( child, highlighted )` would be the way to go, to provide a way of requesting highlight of individual tabs, whether current or not.

Let's leave this till last - we can mull it over while taking care of the rest.